docs/dev/run-cvo-locally.md

```diff
@@ -0,0 +1,51 @@
+# How to Build and Run CVO Locally
+
+This document provides the instructions to build and run the CVO executable locally. Local execution of CVO, for example executing CVO on your development laptop, allows a "quick-turn" test capability whereby the CVO executable can be quickly rebuilt with changes and launched again without the need of starting a new OCP cluster.
+
+## Preconditions
+
+- Build the CVO executable using the standard procedure:
+
+```console
+$ cd /home/jottofar/go/src/github.com/openshift/cluster-version-operator
```

Comment:
nit: `/home/jottofar/` -> `~/`

Reply:
Fixed in [c7cdbc3](https://github.com/openshift/cluster-version-operator/pull/381/commits/c7cdbc37991684945abd08b2e4bcc7bfa574f71d)